### PR TITLE
beta preparation

### PR DESCRIPTION
**Pull request by MartinNowak**
- changelog for 2.069.0
- add beta channel to download page

### CODE REVIEW

**Comment by MartinNowak** [comment]
Looks like the dmd repo of your tester is corrupted @CyberShadow.

**Comment by CyberShadow** [comment]
Should be fixed now, just do a force-push

**Comment by CyberShadow** [comment]
I don't know how much of this is currently considered normal, but...

- "Download D 2.069.0" - link doesn't work
- "released Oct 18, 2015" - date is in the future but uses past tense
- no clear indication in the changelog that this is not a final release

**Comment by MartinNowak** [comment]
Updated

**Comment by MartinNowak** [comment]
Anything left to do @CyberShadow?

File: changelog/changelog.ddoc [hunk 1]
```diff
@@ -12,13 +12,22 @@ SEARCH_OPTIONS_EXTRA=<option value="dlang.org/changelog" selected>Change
...
 $(SMALL released $1, $2)
 )
 $4
-</div>
+)
+
+BETA_VERSION=
+$(DIVC version,
+$(P
+$(B $(LARGE $(LINK2 http://downloads.dlang.org/pre-releases/2.x/$(VER), Download D $(VER))))$(BR)
```
Comment (CyberShadow) [inline changelog/changelog.ddoc]: Could you please add "Beta" here too? Otherwise LGTM :)

Comment (MartinNowak) [inline changelog/changelog.ddoc]: > Could you please add "Beta" here too? Otherwise LGTM :)

Done

**Comment by CyberShadow** [comment]
Awesome

**Comment by MartinNowak** [comment]
> Awesome

Now we only need the nightly channel ;).

**Comment by MartinNowak** [comment]
For reference, I used inline-block with the word-spacing hack instead of a table b/c it allows a vertical layout on mobile.

**Comment by CyberShadow** [comment]
Ah, cool.

**Pull request merged**